shared/js/background/trackers.es6.js

```diff
@@ -31,6 +31,7 @@ function loadLists () {
 function isTracker (urlToCheck, thisTab, request) {
     let currLocation = thisTab.url || ''
     let siteDomain = thisTab.site ? thisTab.site.domain : ''
+    let abbrUrl = urlToCheck.slice(0, urlToCheck.indexOf('?')).substring(0, 75)
```

Comment:
Some tracking request urls are ridiculously long and contain very slightly different params, so I'm abbreviating them here by dropping everything after the ?, then limiting to 75 chars. This should help to reduce the total number of calls made for the similar tracking requests (eg pixels of the form domain.com/pixel.gif?tracking-params).